tests/Responses/Expected/product_list_page_by_wood_material_and_en_GB_locale_using_search_query.json

```diff
@@ -0,0 +1,354 @@
+{
+  "page":1,
```

Comment:
This file has a different indentation than others. Also, this will make test diff even less readable.